proto/src/determined/api/v1/experiment.proto

```diff
@@ -60,12 +60,14 @@ message GetExperimentsRequest {
     int32 limit = 4;
     // Limit experiments to those that match the description.
     string description = 5;
+    // Limit experiments to those that match the provided labels.
+    repeated string labels = 6;
```

Comment:
This seems like an unnecessary breakage if we have already announced this API...?

Reply:
hmm, we're adding an optional parameter what would be breaking? this is requested/needed to fully utilize the pagination on the new endpoint. for the same use cases the old one had

Reply:
if you're talking about the ordering of the parameters that'd only come into play for gRPC use cases which we don't have and have not announced.

Reply:
ok, cool